static/templates/admin_user_list.handlebars

```diff
@@ -34,8 +34,30 @@
         {{/if}}
       {{/if}}
     </span>
+    {{#if is_active}}
+      <button class="button btn-primary open-user-form" title="{{t 'Edit User' }}" data-email="{{email}}">
+          <i class="icon-vector-pencil"></i>
+      </button>
+    {{/if}}
     <div class='admin-user-status'>
     </div>
   </td>
 </tr>
+{{#if is_active}}
+<tr class="user-name-form display-none" id="user_form_{{email}}">
+  <td colspan="{{#if is_bot}}4{{else}}3{{/if}}">
+    <form class="form-horizontal name-setting">
+      <input type="hidden" name="is_full_name" value="true" />
+      <div class="input-group" id="name_change_container">
+        <label for="full_name">{{t "Full name" }}</label>
+        <input type="text" name="full_name" id="full_name" value="{{ full_name }}" />
+      </div>
+      <div class="input-group">
+        <input type="button" class="btn btn-primary submit_name_changes" value="Save Changes" />
+        <button type="button" class="btn btn-default reset_edit_user">Cancel</button>
+      </div>
+    </form>
+  </td>
+</tr>
+{{/if}}
```

Comment:
This looks good, but I just merged a refactoring that split this file apart, so you'll need to resolve the merge conflicts.